easybuild/easyconfigs/s/SAGECal/SAGECal-0.8.4-foss-2023b.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'CMakeMake'
+
+name = 'SAGECal'
+version = '0.8.4'
+
+homepage = 'https://sagecal.sourceforge.net/'
+description = """
+SAGECal is a fast, distributed and GPU accelerated radio astronomial
+calibration package. The many optimization algorithms in SAGECal are
+implemented in a computationally efficient way and can be used in many
+other applications.
+"""
+
+toolchain = {'name': 'foss', 'version': '2023b'}
+
+source_urls = [GITHUB_LOWER_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['73a71e1c1a090aa0e61746066846bfb41b28c92c520f513b5456a8e629365c14']
+github_account = 'nlesc-dirac'
+
+builddependencies = [
+    ('binutils', '2.40'),
+    ('CMake', '3.27.6'),
+    ('pkgconf', '2.0.3'),
+]
+
+dependencies = [
+    ('CFITSIO', '4.3.1'),
+    ('WCSLIB', '7.11'),
+    ('GLib', '2.78.1'),
+    # needed are libcasa-casa4, casacore-dev, casacore-data, casacore-tools, not sure if this provides all?
```

Comment:
I've installed these packages in an Ubuntu container and compared the contents with an EB-installed `casacore`, and the latter should provide everything. So I think we can get rid of the comment (or at least the question)?
```suggestion
```